### PR TITLE
CAPA: Release v26.1.2.

### PR DESCRIPTION
**Pull request by fiunchinho**
Towards https://github.com/giantswarm/roadmap/issues/3676

### Checklist

- [ ] Roadmap issue created
- [ ] Release uses latest stable Flatcar
- [ ] Release uses latest Kubernetes patch version

### Triggering E2E tests

To trigger the E2E test for each new Release added in this PR, add a comment with the following:

`/run releases-test-suites`

If you want to trigger conformance tests, you can do so by adding a comment similar to the following:

`/run conformance-tests PROVIDER=capa RELEASE_VERSION=29.1.0`

For more details see the [README.md](/README.md#running-tests-against-prs).

### CODE REVIEW

File: capa/v26.1.2/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# :zap: Giant Swarm Release v26.1.2 for  :zap:
```
Comment (Gacko) [inline capa/v26.1.2/README.md:1]:
```suggestion
# :zap: Giant Swarm Release v26.1.2 for CAPA :zap:
```

File: capa/v26.1.2/announcement.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+**Workload cluster release v25.1.2 for CAPA is available**. This release allows to have different node pools with different configuration to support legacy cgroupsv1.
```
Comment (Gacko) [inline capa/v26.1.2/announcement.md:1]:
```suggestion
**Workload cluster release v26.1.2 for CAPA is available**. This release allows to have different node pools with different configuration to support legacy cgroupsv1.
```

**Review by Gacko — CHANGES_REQUESTED** [review]
Looks very good, thank you very much! Just two very small nits.

We should definitely improve `devctl`, so this gets easier. 🙂

**Review by Gacko — APPROVED** [review]

**Comment by fiunchinho** [comment]
Tests are failing because ec2 instances won't come up due to
```
failed to create AWSMachine instance: failed to find ami: found no AMIs
      with the name: "flatcar-stable-3815.2.5-kube-1.26.15-tooling-N/A-gs"
```

**Comment by Gacko** [comment]
Oh, damn. This is my fault... I think `cluster-aws` v1.3.3 was forged for CAPA v27.2.0 because of the private ECR thingy. And for that to work, we needed the new image lookup format with `os-tooling`, which then got introduced in `cluster` v1.0.2.

This means we will need to build images for Kubernetes v1.25 using the new image name format. Gonna trigger them.

For consistency reasons we should probably also do a v25.2.2 then, right?

**Review by Gacko — CHANGES_REQUESTED** [review]
Ah, damn, with the latest changes in image lookup format the nodes will roll. So we need a v26.2.0.

**Comment by Gacko** [comment]
Here's a PR for a v26.2.0: https://github.com/giantswarm/releases/pull/1446

**Comment by fiunchinho** [comment]
Closing this one then. Thanks.

**Pull request closed without merging**